spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  describe "Factory" do
+    it "should have a valid factory" do
+      expect(FactoryBot.create(:user)).to be_valid
```

Comment:
I see that you included `FactoryBot::Syntax::Methods` in your RSpec configuration. That is really good. Noww, make sure you use those methods by shortening this line to:
``` 
expect(create(:user)).to be_valid
```